### PR TITLE
Include group publishing link in "Register as a chrome web store developer" page

### PR DESCRIPTION
**Pull request by uayyagari**
Changes proposed in this pull request:

- Include group publishing link in "Register as a chrome web store developer" page

### CODE REVIEW

**Comment by uayyagari** [comment]
@googlebot I signed it!

File: site/en/docs/webstore/register/index.md [hunk 1]
```diff
@@ -31,6 +31,6 @@ screen appears as shown here:
 Once you pay the registration fee and agree to the terms, you will not see this registration page
 again.
 
-<div class="aside aside--note"><strong>Group publishers</strong> note that only members of the group who've paid the registration fee can publish updates.</div>
+<div class="aside aside--note"><a href="https://developer.chrome.com/docs/webstore/group-publishers/"><strong>Group publishers</strong></a> note that only members of the group who've paid the registration fee can publish updates.</div>
```
Comment (awfuchs) [inline site/en/docs/webstore/register/index.md:34]: The target for this should be a path from the base:

<a href="/docs/webstore/group-publishers/">

Comment (awfuchs) [reply site/en/docs/webstore/register/index.md:34]: Other than that, looks great!

**Review by awfuchs — DISMISSED** [review]

File: site/en/docs/webstore/register/index.md (hunk 1 above)
Comment (dotproto) [inline site/en/docs/webstore/register/index.md:34]:
```suggestion
<div class="aside aside--note"><a href="/docs/webstore/group-publishers/"><strong>Group publishers</strong></a> note that only members of the group who've paid the registration fee can publish updates.</div>
```

**Review by dotproto — APPROVED** [review]
Thanks for adding this link!

**Pull request merged**